public/javascripts/running.js

```diff
@@ -168,39 +148,34 @@ function enableInteractive() {
 }
 
 function disableInteractive() {
-    window.hide("stop_button");
+    $("#stop_button").hide();
     sendCommand("disable_interactive_mode");
 }
 
 function sendCommand(command) {
-    var wid = window.testStatus.workerid;
+    var wid = testStatus.workerid;
```

Comment:
I will do the same for the needle editor pull request then and do the general camelCasing in both routes and kill the rest of prototype with it